### PR TITLE
Fixed Quote Author's Name Issue

### LINKED ISSUE #631: Quote author not visible
### Describe the bug
When on the homepage (not logged in), @bdougie's name on his quote is not visible on Firefox. 

### To Reproduce
1. Log out
1. See the quotes

### Expected behavior
The quotes look the same in all browsers.

### Screenshots
Chrome:
chrome
Firefox:
firefox

### Desktop

- OS: Windows 10
- Browser: Google Chrome Beta, Firefox Developer Edition, Tor Browser
- Version: latest

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Release

Fixed #631 

In `Chrome` also: Auther's Name going outside of the flexbox

25p

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

After Changing 
`Firefox`:

after

`Chrome` :

chrome

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
Hi @abhisheknaiidu, thanks so much for taking this on. Please feel free to jump into the discord if you have questions about the project. 

Rather than the fontSize addition, I would recommend adding paddingBottom to the .quotes box. Are you able to update that here: https://github.com/open-sauced/open-sauced/blob/master/src/styles/Header/Hero.js#L178-L180.

File: src/components/Hero.js [hunk 1]
```diff
@@ -71,14 +71,14 @@ function Hero({handleLogIn}) {
           <Flex className="quotes">
             <Flex className="quote">
               <img alt="open sauced" src={gucci} />
-              <span>
+              <span style={{fontSize: 18}}>
```
Comment (bdougie) [inline src/components/Hero.js:74]: Not need, per my comment on the PR. Instead, use extra padding. Be sure to use the variabls.js. 

```css
.quotes {
  padding-bottom :${size.tiny}
  ...
}
```

**Comment by abhisheknaiidu** [comment]
@bdougie Yea I have changed as per you said

**Comment by bdougie** [comment]
Welcome to the Pizza Party @abhisheknaiidu

**Pull request merged**